src/App.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    key: '',
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.handleKeyPressed);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handleKeyPressed);
+  }
+
+  handleKeyPressed = (event: KeyboardEvent) => {
+    this.setState({
+      key: event.key,
+    });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        {this.state.key !== ''
+          ? (
+            <p className="App__message">
+              The last pressed key is [
+              {this.state.key}
+              ]
+            </p>
+          )
+          : (
+            <p className="App__message">
+              Nothing was pressed yet
+            </p>
```

Comment:
You can also use ternary just for content and no need to duplicate p tag with class ` App__message`